asreview/io/utils.py

```diff
@@ -129,7 +129,7 @@ def standardize_dataframe(df, column_spec={}):
         logging.warning("Unable to detect titles in dataset.")
 
     # Replace NA values with empty strings.
-    for col in ["title", "abstract", "authors", "keywords"]:
+    for col in ["title", "abstract", "authors", "keywords", "notes"]:
```

Comment:
To be sure, if `notes` is an empty string, then there is nothing written to the RIS. Or is there an empty tag written?

Reply:
To verify, nothing is written to the RIS export. The empty string is just to avoid NA value for "notes".
```
(base) ottomattas@Ottos-MacBook-Pro rispy % python3
Python 3.9.6 | packaged by conda-forge | (default, Jul 11 2021, 03:35:11) 
[Clang 11.1.0 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rispy
>>> entries = [{'type_of_reference': 'JOUR','id': '42','notes':'','abstract': 'Lorem ipsum'}]
>>> filepath = 'export.ris'
>>> with open(filepath, 'w') as bibliography_file: rispy.dump(entries, bibliography_file)
... 
>>> 
(base) ottomattas@Ottos-MacBook-Pro rispy % cat export.ris 
1.
TY  - JOUR
ID  - 42
AB  - Lorem ipsum
ER  - 
```